src/lints/unwind_function_missing.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "unwind_function_missing",
+    human_readable_name: "unwind function missing",
+    description: "pub fn changed from an unwind-capable ABI to the same-named ABI without unwind: \"C-unwind\" to \"C\"",
+    required_update: Major,
+    reference_link: None,
```

Comment:
Please include an appropriate reference link here. Anything is better than nothing. For example, we could link to the reference page that other ABI-related lints use, if there's nothing more specific.

Reply:
Attached the [proposal](https://rust-lang.github.io/rfcs/2945-c-unwind-abi.html) to add `C-unwind` ABI. It contains lots of information about the design and behaviours. It also links to the RFC PR and issue to `C-unwind`.